alexey.fesenko/task_2.rb

```diff
@@ -0,0 +1,3 @@
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+arr = hash.select { |k| k.to_s.downcase.start_with? "yes" }.keys
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals
Single quotes uses a little less memory then double quotes. So it's better to use '' instead of "".